README.md

```diff
@@ -1,3 +1,124 @@
 # rpi-chromium-mods
 
 This is our "fork" of Raspberry Pi's `rpi-chromium-mods` debian repository.
+
+**Last Updated: `Kano OS v4.3.0`**
+
+
+## Updating rpi-chromium-mods
+
+In Kano OS, we mirror Rasbian and Raspberry Pi debian sources. Therefore, when
+we update the Raspberry Pi mirrors, we get a new version of the original
+rpi-chromium-mods. This newer version will always be overshadowed by Kano's
+`rpi-chromium-mods-kano`.
+
+As such, in order to update the package, you'll simply need to unpack the newer
+version of rpi-chromium-mods and add the files to this repository.
+
+
+## Commit Conventions
+
+From 5 April 2019, the contributions to this project will follow these rules:
+
+  - [Our single commit which adds everything on top of a default version]
```

Comment:
If we want to add more changes into this package, do we modify this base commit or do we stack on top?

Reply:
The problem I have with multiple commits is that I find them a bit unwieldy in the long run. Suppose you have 20 commits on top of the last `[Base version of rpi-chromium-mods]`. When you insert another base version under those 20, you might find yourself solving merge conflicts forever.

Reply:
I guess debian use `quilt` for this sort of thing: https://raphaelhertzog.com/2012/08/08/how-to-use-quilt-to-manage-patches-in-debian-packages/

Reply:
Oh totally! Definitely worth having a look at `quilt`.

Reply:
I've had a quick glance and it's definitely interesting. I think though going from Zero to Hero we'd need to spend a bit of time playing with it. In the spirit of iteration, I've created `OS-954` for us to look at this more in depth.

Reply:
Yeah, not suggesting `quilt` for this PR. Take your time to look into it, some people hate it :-)

Reply:
👍